packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/index.tsx

```diff
@@ -1,9 +1,20 @@
 import React from 'react'
+import { connect, ConnectedProps } from 'react-redux'
+import { withRouter } from 'react-router-dom'
+import { bindActionCreators, Dispatch } from 'redux'
+
+import { actions } from 'data'
 
 import DebitCard from './template'
 
-const DebitCardContainer = () => {
-  return <DebitCard />
-}
+const DebitCardContainer = (props: Props) => <DebitCard {...props} />
```

Comment:
Could you destructure these props?

Reply:
I don't see the point making destructuring here when I just pass all of them to other components